fe/fe-core/src/main/java/com/starrocks/qe/QeProcessorImpl.java

```diff
@@ -185,8 +185,11 @@ public TReportExecStatusResult reportExecStatus(TReportExecStatusParams params,
         final TReportExecStatusResult result = new TReportExecStatusResult();
         final QueryInfo info = coordinatorMap.get(params.query_id);
         if (info == null) {
-            LOG.info("ReportExecStatus() failed, query does not exist, fragment_instance_id={}, query_id={},",
-                    DebugUtil.printId(params.fragment_instance_id), DebugUtil.printId(params.query_id));
+            // ignore log query
```

Comment:
Can you elaborate more on the reason why you ignored the query execute report in the comment?

Reply:
case： not enable profile
instances  transfer from bottom to top by chunk,  fe get result from result sink, will call deregister, remove form cood,  but report exec status not sent sequentially